examples/metrics-basic/src/main.rs

```diff
@@ -1,32 +1,34 @@
+use opentelemetry::global::{meter_provider, set_meter_provider, shutdown_meter_provider};
 use opentelemetry::metrics::Unit;
 use opentelemetry::{metrics::MeterProvider as _, KeyValue};
 use opentelemetry_sdk::metrics::{PeriodicReader, SdkMeterProvider};
 use opentelemetry_sdk::{runtime, Resource};
 use std::error::Error;
 
-fn init_meter_provider() -> SdkMeterProvider {
+fn init_meter_provider() {
     let exporter = opentelemetry_stdout::MetricsExporterBuilder::default()
         // uncomment the below lines to pretty print output.
         //  .with_encoder(|writer, data|
         //    Ok(serde_json::to_writer_pretty(writer, &data).unwrap()))
         .build();
     let reader = PeriodicReader::builder(exporter, runtime::Tokio).build();
-    SdkMeterProvider::builder()
+    let provider = SdkMeterProvider::builder()
         .with_reader(reader)
         .with_resource(Resource::new(vec![KeyValue::new(
             "service.name",
             "metrics-basic-example",
         )]))
-        .build()
+        .build();
+    set_meter_provider(provider);
```

Comment:
```suggestion
    opentelemetry::global::set_meter_provider(provider);
```
or 
```
global::set_meter_provider(provider);
```

I think these versions makes it easier to convey the intent?